### DISCUSSION TITLE
Bug report: Files viewed filter removes files instead of showing them

### DISCUSSION THREAD

**Discussion by tiagojdf** [community/community#8662] [Pull Requests]
Recently the behaviour of the `Files viewed` filter changed. Before, unchecking it would open viewed files, now it makes them disappear. 
![Screenshot 2021-12-10 at 15 25 02](https://user-images.githubusercontent.com/1483616/145589092-6f0745c4-96b1-4583-ae3d-6091b8b4e5d3.png)

Maybe that's the new intended behaviour, but it prevents a previous use case. When reviewing a PR, I mark the files reviewed as I go. If for some reason I need to go back and see how a declaration was made in one of those files, I would disable viewed files so I could then use search to find the code. Now, since it just removes the files (which doesn't make much sense in my opinion since they are already minimised), it makes it impossible to quickly search bot viewed and not viewed files. I need to go hand by hand looking for the right file that had the change I wanted.

**Comment by MathiasReker** [comment]
Hello. You can report the bug here: https://support.github.com/contact/bug-report

**Comment by tiagojdf** [comment]
Done. Thanks for your answer
https://support.github.com/ticket/personal/0/1694605

**Reply by MathiasReker** [reply]
You are welcome :)
